lib/msf/core/exploit/smb/client.rb

```diff
@@ -16,6 +16,8 @@ module Client
       include Msf::Exploit::Remote::Tcp
       include Msf::Exploit::Remote::NTLM::Client
 
+      CONST  = Rex::Proto::SMB::Constants
```

Comment:
Looking at where this is consumed, a shorthand like this probably belongs in the class it is consumed in.  I would suggest moving to ms17_010_psexec.rb directly

Note I have checked the other 3 modules that currently use this mixin and if they use this value they define the shorthand locally.

Reply:
Yeah, XCEPT gets used in more places than I originally found. SIMPLE probably too. I'll just add all 3 back for now.